src/util-inl.h

```diff
@@ -198,6 +199,14 @@ TypeName* Unwrap(v8::Local<v8::Object> object) {
   return static_cast<TypeName*>(pointer);
 }
 
+void SwapBytes(uint16_t* dst,
```

Comment:
Could you make a more generic template function to work for types with 4 bytes and 8 bytes?